### PR TITLE
feat: document `--gas.price` flag

### PR DESCRIPTION
**Pull request by jcstein**
## Overview

This documents https://github.com/celestiaorg/celestia-node/pull/3094

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

## Summary by CodeRabbit

- **Documentation**
	- Updated the tutorial on setting gas prices in the `celestia` CLI. The `--fee` and `--gas.limit` flags have been replaced by a single `--gas-price` flag. Gas limit calculation now depends on submitted data size.

### CODE REVIEW

File: developers/node-tutorial.md [hunk 1]
```diff
@@ -775,32 +775,23 @@ Which will return:
...
-
-```bash
-celestia blob submit 0x42690c204d39600fddd3 'gm' --gas.limit 100000 \
-  --node.store $NODE_STORE
-```
+Learn [more about gas fees and limits](../developers/submit-data.md).
 
-To set a fee of 10000 utia and gas limit of 100000, use the
-`--fee 10000 --gas.limit 100000` flags:
+To set a higher gas price of 0.004 utia, use the `--gas-price 0.002` flag:
```
Comment (jcstein) [inline developers/node-tutorial.md:791]:
```suggestion
To set a higher gas price of 0.004 utia, use the `--gas-price 0.004` flag:
```

**Comment by jcstein** [comment]
however, this shouldn't be merged until we've upgraded Arabica/Mocha/Mainnet IMO cc @Bidon15 for vis
https://github.com/celestiaorg/docs/pull/1404

File: developers/node-tutorial.md [hunk 2]
```diff
@@ -775,32 +775,23 @@ Which will return:
 }
 ```
 
-### Setting the gas fee and limit
+### Setting the gas price
 
-To set the gas fee and limit, you can use the `--fee` and `--gas.limit` flags
-respectively when submitting data using the RPC CLI.
+The `--gas-price` flag allows you to specify the gas price for the
```
Comment (jcstein) [inline developers/node-tutorial.md:780]:
```suggestion
The `--gas.price` flag allows you to specify the gas price for the
```

File: developers/node-tutorial.md [hunk 3]
```diff
@@ -775,32 +775,23 @@ Which will return:
...
 
-Learn [more about gas fees and limits](../developers/submit-data.md).
-
-To set the fee of 10000 utia, use the `--fee 10000` flag:
-
-```bash
-celestia blob submit 0x42690c204d39600fddd3 'gm' --fee 10000 \
-  --node.store $NODE_STORE
-```
+To set the gas price, you can use the `--gas-price` flag.
```
Comment (jcstein) [inline developers/node-tutorial.md:785]:
```suggestion
To set the gas price, you can use the `--gas.price` flag.
```

File: developers/node-tutorial.md [hunk 4]
```diff
@@ -775,32 +775,23 @@ Which will return:
...
-```
+Learn [more about gas fees and limits](../developers/submit-data.md).
 
-To set a fee of 10000 utia and gas limit of 100000, use the
-`--fee 10000 --gas.limit 100000` flags:
+To set a higher gas price of 0.004 utia, use the `--gas-price 0.004` flag:
 
 ```bash
-celestia blob submit 0x42690c204d39600fddd3 'gm' --fee 10000 --gas.limit 100000 \
+celestia blob submit 0x42690c204d39600fddd3 'gm' --gas-price 0.004 \
```
Comment (jcstein) [inline developers/node-tutorial.md:794]:
```suggestion
celestia blob submit 0x42690c204d39600fddd3 'gm' --gas.price 0.004 \
```

File: developers/node-tutorial.md [hunk 5]
```diff
@@ -775,32 +775,23 @@ Which will return:
...
-
-```bash
-celestia blob submit 0x42690c204d39600fddd3 'gm' --gas.limit 100000 \
-  --node.store $NODE_STORE
-```
+Learn [more about gas fees and limits](../developers/submit-data.md).
 
-To set a fee of 10000 utia and gas limit of 100000, use the
-`--fee 10000 --gas.limit 100000` flags:
+To set a higher gas price of 0.004 utia, use the `--gas-price 0.004` flag:
```
Comment (jcstein) [inline developers/node-tutorial.md:791]:
```suggestion
To set a higher gas price of 0.004 utia, use the `--gas.price 0.004` flag:
```

**Review by YazzyYaz — APPROVED** [review]

**Pull request merged**